programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -839,13 +839,7 @@ static int run_tunnel(uv_loop_t *ziti_loop, uint32_t tun_ip, uint32_t dns_ip, co
     }
 
 #if _WIN32
-    const wchar_t* tun_name = get_tun_name(tun->handle);
-    size_t tun_name_len = wcslen(tun_name);
-    char* name = calloc(tun_name_len, sizeof(char) + 1);
-    wcstombs(name, tun_name, tun_name_len + 1);
-    set_tun_name(name); //sets the tunnel status, tun name...
```

Comment:
do we still want to set `tnl_status.TunName`?

Reply:
gah. yes. i meant to put that back. thank you for reminding me!